_redirects

```diff
@@ -1,3 +1,4 @@
+/faqs/Accounts/delete-users /faqs/Accounts/add-delete-users 301
```

Comment:
@mirnawong1 Instead of this being an FAQ, and now that we have a section for it, how about we put it in "Manage access" maybe under "User access and licenses? I think it might be more discoverable there!

https://docs.getdbt.com/docs/cloud/manage-access/about-user-access

Reply:
Great shout @runleonarun ! I was thinking of that originally but then I saw Matt's faq on how to delete users, so then thought I'd repurpose the same faq to add and delete users.

I'll do that tomorrow because I agree it should be more discoverable!